akalisten/templates/lotsude/style.css

```diff
@@ -51,6 +53,10 @@ li {
     box-shadow: 0 1px 3px rgba(0,0,0,0.1);
     transition: background-color 0.3s;
 }
+.multi-column-list {
+    column-count: 3;
```

Comment:
I really like the columns. I'm just worried because the location information and probably also other information can become very long. This breaks the layout for me.

![image](https://github.com/user-attachments/assets/a10c2419-253b-4633-8a15-73a274645fb1)

Reply:
mh, the element-break is not nice indeed. Thank for the finding.
`break-inside` improves it. Not perfect, but suffcient IMHO.

![image](https://github.com/user-attachments/assets/7b6eb5ea-2f48-4bb4-baeb-6cfe9e288a4c)

![image](https://github.com/user-attachments/assets/594fe373-ce11-4814-be34-3ac6c8dd21d2)